### PR TITLE
More wheels: manylinux and MacOS

### PR DESCRIPTION
**Pull request by mgedmin**

### CODE REVIEW

**Comment by mgedmin** [comment]
Uh, the Mac OS Python 3.5 build fails because it tries to pull in zope.security, which setup_requires zope.proxy.

Do we have a dependency loop?  Why do zope.proxy builds need zope.security?

**Comment by mgedmin** [comment]
> Do we have a dependency loop?

Yes: https://github.com/zopefoundation/zope.proxy/blob/10b7f39aede02872a9c9c7803127a48f90625ef9/setup.py#L130-L132

**Comment by mgedmin** [comment]
Ok, build fixed, all's well!

File: .travis.yml [hunk 1]
```diff
@@ -1,35 +1,104 @@
...
+  - python setup.py bdist_wheel
 
 after_success:
-    - coveralls
+  - coveralls
+  - |
+    if [[ $TRAVIS_TAG && "$TRAVIS_OS_NAME" == "osx" ]]; then
+      # macpython 3.5 doesn't support recent TLS protocols which causes twine
+      # upload to fail, so we use the system Python to run twine
+      /usr/bin/python -m pip install twine
```
Comment (mgedmin) [inline .travis.yml]: This needs a `/usr/bin/python -m ensurepip` first.

Comment (mgedmin) [reply .travis.yml]: Actually it needs much more, see https://github.com/zopefoundation/zope.interface/pull/148.  (I'd prefer to release zope.interface 1.7.1 to make sure that recipe works before copying it elsewhere.)

**Comment by mgedmin** [comment]
Rebased on master, fixed Mac OS twine installation.

File: .travis.yml [hunk 2]
```diff
@@ -14,19 +21,88 @@ jobs:
...
+  # NB: let's install . (zope.proxy) first separately, because we have a nasty
+  # dependency loop: .[test] wants zope.security that setup_requires zope.proxy
+  # (and setup_requires is broken on Mac OS Python 3.5 due to TLS version
+  # compatibility issues that pip knows how to work around, but setuptools
+  # don't).
+  - pip install -e .
   - pip install -e .[test,docs]
 
 script:
+  - python --version
```
Comment (jamadden) [inline .travis.yml]: 👍

**Pull request merged**